tests/test_algorithm.py

```diff
@@ -197,12 +163,27 @@
 _multiprocess_can_split_ = False
 
 
-class TestRecordAlgorithm(WithSimParams, WithDataPortal, ZiplineTestCase):
-    ASSET_FINDER_EQUITY_SIDS = 133,
+class TestRecord(zf.WithMakeAlgo, zf.ZiplineTestCase):
+    ASSET_FINDER_EQUITY_SIDS = (133,)
+    SIM_PARAMS_DATA_FREQUENCY = 'daily'
+    DATA_PORTAL_USE_MINUTE_DATA = False
 
     def test_record_incr(self):
-        algo = RecordAlgorithm(sim_params=self.sim_params, env=self.env)
-        output = algo.run(self.data_portal)
+
```

Comment:
A general theme of this PR is that I tried to "inline" algorithms into test sites if they were small and if there were assertions in the test body that depended on the behavior of the algorithm.

For tests that were just "run algorithm X", I didn't see as much value in doing the inlining.

Reply:
Thank you. I agree with the inlining. While debugging (or just reading tests), this will reduce cognitive load.